client/my-sites/site-settings/sso.jsx

```diff
@@ -49,7 +49,7 @@ const Sso = ( {
 					<JetpackModuleToggle
 						siteId={ selectedSiteId }
 						moduleSlug="sso"
-						label={ translate( 'Allow sign in using WordPress.com accounts' ) }
+						label={ translate( 'Allow users to log in to this site using WordPress.com accounts' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **14** times:
`translate( 'Allow your users to log in using their WordPress.com accounts.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).